docs/index.rst

```diff
@@ -346,6 +346,24 @@ Note the double asterisk, which indicates any subdirectory included in the paren
 By default, the script will perform a dry run and the files will not be moved. You need to run
 ``move_files.py`` with the option ``--dry False`` to move them.
 
+Sanity check
+----------
+The script to perform certain checks on archived files. 
+
+To compare the files between the staging and archive folders,
+you need to run the script doing::
+
+  sanitycheck_on_dbfiles.py --settings settings.ini --dbpwd $DBPWD --dbname $DBNAME --directory dir_name
+
+- ``--directory`` is the directory in the FTP area to compare staging and archive.
```

Comment:
To add this to the docs, 
if --directory is not called, --firepwd needs to be used and that checks all the files in the firepath, both can be used but --firepwd is not necessary when --directory is used. One of them option needs to be used at the moment except if the code is changed above.